cypress/support/step_definitions/annual-review/scheduler/there_is_a_list_item_unpublished_automatically_over_a_year_ago.js

```diff
@@ -0,0 +1,38 @@
+Given("there is a list item unpublished automatically over a year ago", () => {
+  cy.task("db", {
+    operation: "listItem.create",
+    variables: {
+      data: {
+        type: "lawyers",
```

Comment:
I believe listItems also have a reference column. You can use that if it helps you locate the entry easier? e.g. `reference: AUTO_DELETE`, then you can just do `listItem.find({where: { reference: "AUTO_DELETE" }})`